pom.xml

```
@@ -822,5 +822,27 @@ THE SOFTWARE.
         <failIfNoTests>false</failIfNoTests>
       </properties>
     </profile>
+    <profile>
+      <id>jdk-17-and-above</id>
+      <activation>
+        <jdk>[17,)</jdk>
+      </activation>
+      <properties>
+        <!--
+          Animal Sniffer does not work on Java 17, but we cannot remove it from our configuration
+          entirely until we drop support for Java 8. As a temporary solution, we skip it when
+          running on Java 17. When JENKINS-54842 is resolved, this can be deleted.
```

```suggestion
          running on Java 17. When https://github.com/jenkinsci/pom/pull/209 is incorporated, this can be deleted.
```
for https://github.com/jenkinsci/pom/pull/209?

---

Maybe. I am thinking about doing a more robust implementation of that.